steward/src/commands/allow_erc20.rs

```diff
@@ -36,7 +36,7 @@ impl Runnable for AllowERC20 {
 
         let name = &config
             .keys
-            .rebalancer_key
+            .delegate_key
```

Comment:
Tagging this file as meant to be deleted based on `AllowErc20` command being removed.